client/state/site-settings/selectors.js

```diff
@@ -44,7 +44,7 @@ export function getSiteSettingsSaveRequestStatus( state, siteId ) {
  * @return {Object}        Site settings
  */
 export function getSiteSettings( state, siteId ) {
-	return get( state.siteSettings.items, [ siteId ], null );
+	return get( state, [ 'siteSettings', 'items', siteId ], null );
```

Comment:
Updated in https://github.com/Automattic/wp-calypso/pull/13329/commits/aed545cd12119fe78ae841e829652b8d02aba6c0 and removed the commit that altered `getSiteSettings`.